lib/net/scp.rb

```diff
@@ -344,11 +344,15 @@ def scp_command(mode, options)
       def start_command(mode, local, remote, options={}, &callback)
         session.open_channel do |channel|
 
-          if options[:shell]
-            escaped_file = shellescape(remote).gsub(/'/) { |m| "'\\''" }
-            command = "#{options[:shell]} -c '#{scp_command(mode, options)} #{escaped_file}'"
+          if options[:windows_path]
```

Comment:
I assume `:windows_path` is a new option. Does it means that the remote uses windows style encoding?